lib/std/posix.zig

```diff
@@ -5871,8 +5873,7 @@ pub fn res_mkquery(
     q[i + 3] = class;
 
     // Make a reasonably unpredictable id
-    var ts: timespec = undefined;
-    clock_gettime(.REALTIME, &ts) catch {};
+    const ts = clock_gettime(.REALTIME) catch undefined;
```

Comment:
```suggestion
    const ts = clock_gettime(.REALTIME) catch unreachable;
```

I realize you haven't actually changed the semantics of this line, but it was weird before too AFAICT; `CLOCK_REALTIME` is supported everywhere.

Reply:
got it